src/main/java/com/epam/ms/config/RabbitConfiguration.java

```diff
@@ -0,0 +1,51 @@
+package com.epam.ms.config;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.amqp.core.AmqpAdmin;
+import org.springframework.amqp.core.Queue;
+import org.springframework.amqp.rabbit.connection.CachingConnectionFactory;
+import org.springframework.amqp.rabbit.connection.ConnectionFactory;
+import org.springframework.amqp.rabbit.core.RabbitAdmin;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+
+@Slf4j
+@Configuration
+@PropertySource("classpath:application.yml")
```

Comment:
This is redundant, please, try without this annotation

Reply:
Thank you, done